ui/src/androidTest/java/com/hyperwallet/android/transfermethod/ui/ListReceiptsTest.java

```diff
@@ -0,0 +1,150 @@
+
+        onView(withId(R.id.list_receipts))
+                .check(matches(atPosition(0, hasDescendant(withText("November 2017")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(0,
+                hasDescendant(withText(com.hyperwallet.android.receipt.R.string.credit)))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(0, hasDescendant(withText("Payment")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(0, hasDescendant(withText("+ 20.00")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(0, hasDescendant(withText("November 01, 2017")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(0, hasDescendant(withText("USD")))));
+
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(1,
+                hasDescendant(withText(com.hyperwallet.android.receipt.R.string.credit)))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(1, hasDescendant(withText("Payment")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(1, hasDescendant(withText("+ 25.00")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(1, hasDescendant(withText("November 02, 2017")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(1, hasDescendant(withText("CAD")))));
+
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(2,
+                hasDescendant(withText(com.hyperwallet.android.receipt.R.string.debit)))));
+        onView(withId(R.id.list_receipts)).check(
+                matches(atPosition(2, hasDescendant(withText("Card Activation Fee")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(2, hasDescendant(withText("- 1.95")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(2, hasDescendant(withText("November 10, 2017")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(2, hasDescendant(withText("USD")))));
+
+        onView(withId(R.id.list_receipts))
+                .check(matches(atPosition(3, hasDescendant(withText("December 2017")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(3,
+                hasDescendant(withText(com.hyperwallet.android.receipt.R.string.debit)))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(3, hasDescendant(withText("Card Load")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(3, hasDescendant(withText("- 18.05")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(3, hasDescendant(withText("December 01, 2017")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(3, hasDescendant(withText("USD")))));
+
+        onView(withId(R.id.list_receipts)).check(new RecyclerViewCountAssertion(4));
+    }
+
+    @Test
+    public void testListReceipt_userHasSingleReceipt() {
```

Comment:
Can you rename this to testListReceipts_displayCreditTransaction() and add another test case for testListReceipts_displayDebitTransaction()

Reply:
@skoong Done